lib/ansible/plugins/connection/paramiko_ssh.py

```diff
@@ -6,7 +6,7 @@
 
 DOCUMENTATION = """
     author: Ansible Core Team
-    connection: paramiko
+    connection: paramiko_ssh
```

Comment:
this is incorrect, it uses a alias in plugin_loader to show as paramiko, so the name should be kept

Reply:
Sure. (I guess this needs a hard-coded exception in the validator then.)